postgres/server/image.sls

```diff
@@ -10,15 +10,19 @@ include:
 
 # An attempt to start PostgreSQL with `pg_ctl`
 
-postgresql-start:
+postgresql-running:
   cmd.run:
-    - name: pg_ctl -D {{ postgres.data_dir }} -l logfile start
+    - name: {{ postgres.bake_image_run_cmd }}
     - runas: {{ postgres.user }}
     - unless:
       - ps -p $(head -n 1 {{ postgres.data_dir }}/postmaster.pid) 2>/dev/null
     - require:
       - file: postgresql-pg_hba
 
+postgresql-service-reload:
```

Comment:
This was used to trigger configs reloading instead of service restart after ` pg_ident.conf` would be changed.
Clearly should be the dummy state here to satisfy dependency. Please make sure it is **not** referenced with type:
```yaml
      - module: postgresql-service-reload
```

but must be generic requirement (in `postgres/server/init.sls`):
```yaml
      - postgresql-service-reload
```

For some older Salt versions to be happy.

Reply:
Thanks for comment @vitny ! Is really is referenced as
`      - module: postgresql-service-reload

Last commit deals with this issue.

Reply:
:+1: